.travis.yml

```diff
@@ -10,7 +10,7 @@ rvm:
   - 2.3.4
 before_script:
   - bundle exec sidekiq -d -l /tmp/sidekiq.log
-  - bundle exec rails db:setup
+  - bundle exec rails db:setup   
```

Comment:
@little9 is the extra whitespace here intentional?  If not, can you take it out before we merge the PR?